spec/factories/loans_factory.rb

```diff
@@ -162,20 +162,15 @@
       end
     end
 
-    trait :with_copies do
-      after(:create) do |loan|
-        create(:loan, original: loan)
-      end
-    end
```

Comment:
removing this because it never happens in the codebase and it circumvents our duplicator code which is what we want to actually test.